### DISCUSSION TITLE
After setting up a new nextjs app with create next app, next build fails

### DISCUSSION THREAD

**Discussion by me1k** [vercel/next.js#48944] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
macOs Ventura Version 13.2.1
```

### Link to the code that reproduces this issue

-

### To Reproduce

Just create a new create-next app with the latest version and after try immediatly to run `npm run build` . This fails with

### Describe the Bug

uncaughtException Error: Cannot find module 'node:buffer'
Require stack:
- /Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/context.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/sandbox.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/index.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/build/utils.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/build/entries.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/build/index.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/cli/next-build.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/lib/commands.js
- /Users/meikbolender/Playground/physio/node_modules/next/dist/bin/next
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/context.js:28:60)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/context.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/sandbox.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/server/web/sandbox/index.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/build/utils.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/build/entries.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/build/index.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/cli/next-build.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/lib/commands.js',
    '/Users/meikbolender/Playground/physio/node_modules/next/dist/bin/next'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! physio@0.1.0 build: `NODE_ENV=production next build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the physio@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/meikbolender/.npm/_logs/2023-04-28T06_43_45_038Z-debug.log

### Expected Behavior

Freshly created next js project works out of the box

**Comment by icyJoseph** [comment]
Hi,

I can't reproduce this. I even tried to build your project and it works. Could you please include the `next info` output? What Node.js/npm version have you got installed?

**Comment by balazsorban44** [comment]
This sounds like you are on an older version of Node.js that might not be supported. Hard to tell more as there was no reproduction/system info provided.
